memcached-operator/pkg/controller/memcached/memcached_controller.go

```diff
@@ -109,55 +109,56 @@ func (r *ReconcileMemcached) Reconcile(request reconcile.Request) (reconcile.Res
 		return reconcile.Result{}, err
 	}
 
-	// Check if the Deployment already exists, if not create a new one
-	deployment := &appsv1.Deployment{}
```

Comment:
With these change, the sample loses examples over how to use the client to the CRUD operations which IMHO is something import to shows here as an example. 

c/c @joelanford

Reply:
There's still an `r.client.Get` on line 98, an `r.client.List` on line 171, and an `r.client.Status().Update` on line 181. Are those sufficient, or would you prefer more examples of those?